src/Cli/Exporter.cs

```diff
@@ -65,6 +67,7 @@ public static void Export(ExportOptions options, ILogger logger, FileSystemRunti
             }
 
             cancellationTokenSource.Cancel();
+            return isSuccess ? 0 : -1;
```

Comment:
Similar to -1, can we define 0 as a const success code somewhere?

Reply:
Yes, introduced constants folder with CliReturnCode class with two constants CliReturnCode.GENERAL_ERROR and CliReturnCode.SUCCESS